sys/net/include/etx_beaconing.h

```diff
@@ -26,43 +25,78 @@
 extern "C" {
 #endif
 
-//[option|length|ipaddr.|packetcount] with up to 15 ipaddr|packetcount pairs
-// 1 Byte 1 Byte  1 Byte  1 Byte
+/** 
+ * @brief [option|length|ipaddr.|packetcount] with up to 15 ipaddr|packetcount pairs
+ *        | option | length | ipaddr  | packetcount |
+ *        | ----:  | :----: | :----:  | :----:      |
+ *        | 1 Byte | 1 Byte | 1 Byte  | 1 Byte      |
+ */
 #define ETX_BUF_SIZE   (32)
-
 #define ETX_RCV_QUEUE_SIZE     (128)
 
-/*
- * Default 40, should be enough to get all messages for neighbors.
- * In my tests, the maximum count of neighbors was around 30-something
+/**  
+ * @brief defining the number of neighbours 
  */
-#if ENABLE_DEBUG
-#define ETX_MAX_CANDIDATE_NEIGHBORS (15) //Stacksizes are huge in debug mode, so memory is rare
-#else
-#define ETX_MAX_CANDIDATE_NEIGHBORS (40)
+#if ENABLE_DEBUG                                     
+ /**
+  *Stacksizes are huge in debug mode, so memory is rare so we fix it at 15 
+  */
+#define ETX_MAX_CANDIDATE_NEIGHBORS (15)  
+/**
+ *Default 40, should be enough to get all messages for neighbors. 
+ */
+#define ETX_MAX_CANDIDATE_NEIGHBORS (40)             
 #endif
-//ETX Interval parameters
 #define MS  (1000)
 
-/*
- * ETX_INTERVAL
- *
+/**
+ * @{
+ * @name ETX Interval parameters
```

Comment:
Split this documentation block in two parts, one `@name` for the group, and one `@brief` for the `ETX_INTERVAL` define.